README.md

```diff
@@ -29,9 +29,9 @@ Currently `rocket-sentry` includes two integrations:
   - [X] HTTP method
   - [X] GET query string
   - [X] headers
+  - [X] environment
```

Comment:
When I see the word "environment", I tend to think of environment variables.

Let's clarify this, how about

```suggestion
  - [X] environment name (based on Rocket configuration profile)
```